value/value.go

```diff
@@ -334,25 +319,42 @@ func BooleanValue(b bool) Value {
 func (v Value) String() string {
```

Comment:
This code should never be called from a performance-critical path.

Reply:
We serialize pe's to string (to index maps), and pe's can be made of values.